files/default/sql/ddl/3.2.0__initial_tables.sql

```diff
@@ -2276,4 +2276,4 @@ CREATE TABLE IF NOT EXISTS `hdfs_command_execution` (
   KEY `fk_hdfs_file_command_2_idx` (`src_inode_partition_id`,`src_inode_pid`,`src_inode_name`),
   CONSTRAINT `fk_hdfs_file_command_1` FOREIGN KEY (`execution_id`) REFERENCES `executions` (`id`) ON DELETE CASCADE,
   CONSTRAINT `fk_hdfs_file_command_2` FOREIGN KEY (`src_inode_partition_id`,`src_inode_pid`,`src_inode_name`) REFERENCES `hops`.`hdfs_inodes` (`partition_id`, `parent_id`, `name`) ON DELETE CASCADE
-) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
\ No newline at end of file
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
```

Comment:
@vatj - Can you make sure you don't modify the 3.2.0 files? The tool we use to do database migrations compute checksums of the files. Adding removing new lines changes the checksum and creates issues.

Reply:
I'll fix it thanks for pointing it out